examples/decoding/README.txt

```diff
@@ -1,5 +1,5 @@
 
-Decoding / MVPA
----------------
+Decoding / Encoding
+-------------------
 
-Decoding, a.k.a. MVPA or machine learning examples.
+Decoding, encoding, and general machine learning examples.
```

Comment:
you mean google indexing? fair point...if we're interested in search indexing etc, what if we called it "MVPA, decoding, and machine learning"